r2/r2/templates/base.html

```diff
@@ -68,6 +71,64 @@
 
     ${self.head()}
     <script src="//load.sumome.com/" data-sumo-site-id="2115f2f4b9e86f1f2e73c48fb896958ad06ae47bfb80face9329b72cb21fca6c" async="async"></script>
+    <style>
+    @media only screen and (max-width: 1024px)  {
```

Comment:
What do you think about making this a bit smaller (maybe 750px)? In my opinion, the text is too big on some resized desktop windows. After (left) and before (right):

![screenshot from 2016-11-27 20-47-20](https://cloud.githubusercontent.com/assets/1735266/20656510/c7be91ae-b4e2-11e6-96d0-10b400839f39.png)